lib/assets/javascripts/turbograft/remote.coffee

```diff
@@ -51,10 +51,13 @@ class TurboGraft.Remote
 
   createPayload: (form) ->
     if form
-      if form.querySelectorAll("[type='file']").length > 0
-        formData = new FormData(form)
+      $cleanForm = $(form).clone()
+      $cleanForm.querySelectorAll('input[tg-remote-noserialize]').remove()
```

Comment:
Used this suggestion, but combined with @DrewMartin 's suggestion of using the absence of name